src/status_im2/contexts/quo_preview/wallet/wallet_overview.cljs

```diff
@@ -0,0 +1,77 @@
+(ns status-im2.contexts.quo-preview.wallet.wallet-overview
+  (:require [quo2.core :as quo2]
+            [quo2.foundations.colors :as colors]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [status-im2.contexts.quo-preview.preview :as preview]))
+
+(def descriptor
+  [{:label   "State"
+    :key     :state
+    :type    :select
+    :options [{:key   :loading
+               :value "Loading"}
+              {:key   :default
+               :value "Default"}]}
+   {:label   "Time frame"
+    :key     :time-frame
+    :type    :select
+    :options [{:key   :none
+               :value "None"}
+              {:key   :selected
+               :value "Selected"}
+              {:key   :one-week
+               :value "1 Week"}
+              {:key   :one-month
+               :value "1 Month"}
+              {:key   :three-months
+               :value "3 Months"}
+              {:key   :one-year
+               :value "1 Year"}
+              {:key   :all-time
+               :value "All time"}
+              {:key   :custom
+               :value "Custom"}]}
+   {:label   "Metrics"
+    :key     :metrics
+    :type    :select
+    :options [{:key   :none
+               :value "None"}
+              {:key   :positive
+               :value "Positive"}
+              {:key   :negative
+               :value "Negative"}]}])
+
+(defn cool-preview
+  []
+  (let [state (reagent/atom {:state             :default
+                             :time-frame        :one-week
+                             :metrics           :none
+                             :balance           "€0.00"
+                             :date              "20 Nov 2023"
+                             :begin-date        "16 May"
+                             :end-date          "25 May"
+                             :currency-change   "€0.00"
+                             :percentage-change "0.00%"})]
+    (fn []
+      [rn/touchable-without-feedback {:on-press rn/dismiss-keyboard!}
+       [rn/view {:padding-bottom 150}
+        [rn/view {:flex 1}
+         [preview/customizer state descriptor]]
+        [rn/view
+         {:padding-vertical 60
+          :flex-direction   :row
+          :justify-content  :center}
+         [quo2/wallet-overview @state]]]])))
+
+(defn preview-wallet-overview
+  []
+  [rn/view
+   {:background-color (colors/theme-colors colors/white
+                                           colors/neutral-95)
+    :flex             1}
+   [rn/flat-list
```

Comment:
as @smohamedjavid said in some other pr's. Can we move away from this flat list approach? - Imo I do not know why we are using it here. cant' we just render the components in a normal structure? - curious if anyone knows what benefits do we gain by doing this?